designer/server/src/main/scala/pl/touk/nussknacker/ui/api/TestingApiErrorMessages.scala

```diff
@@ -0,0 +1,41 @@
+package pl.touk.nussknacker.ui.api
+
+object TestingApiErrorMessages {
+
+  val noSourcesWithTestDataGeneration = "No sources with test data generation available"
+
+  val noDataGenerated = "No data was generated"
+
+  val noInputRecords = "No input records found"
+
+  def requestedTooManySamplesToGenerate(maxSamples: Int) =
+    s"Too many samples requested, limit is $maxSamples. Please configure 'testDataSettings.maxSamplesCount' to increase the limit"
+
+  def tooManyCharactersGenerated(length: Int, limit: Int) =
+    s"$length characters were generated, limit is $limit. Please configure 'testDataSettings.testDataMaxLength' to increase the limit"
+
+  def tooManyCharactersReceived(length: Int, limit: Int) =
+    s"Received $length characters, limit is $limit. Please configure 'testDataSettings.testDataMaxLength' to increase the limit"
+
+  def tooManySamplesReceived(count: Int, maxSamples: Int) =
+    s"Received $count samples, limit is: $maxSamples. Please configure 'testDataSettings.maxSamplesCount'"
+
+  def recordParsingError(rawTestRecord: String): String = {
+    val trimmedRawTestRecord = rawTestRecord.take(300)
+    if (trimmedRawTestRecord.length < rawTestRecord.length) {
+      s"Could not parse record (shows fragment): '$trimmedRawTestRecord'"
+    } else {
+      s"Could not parse record: '$rawTestRecord'"
+    }
+  }
+
+  def missingSourceForRecord(sourceId: String, recordIndex: Int) =
+    s"Record ${recordIndex + 1} - scenario does not have source id: '$sourceId'"
```

Comment:
It took me a while to understand what this message mean. Let's rephrase it a bit. Some suggestions:
- "Source with id [$sourceId] doesn't exist for record ${recordIndex + 1}"
- "Problem in record  ${recordIndex + 1} detected: Source [$sourceId] doesn't exist"